src/Components/Web.Extensions/src/Microsoft.AspNetCore.Components.Web.Extensions.csproj

```diff
@@ -1,14 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <Description>A collection of Blazor components for the web.</Description>
+    <StaticWebAssetBasePath>_content/Microsoft.AspNetCore.Components.Web.Extensions</StaticWebAssetBasePath>
```

Comment:
Why are we setting this explicitly? This value matches the default value that it will be given AFAIK

Reply:
No reason in particular, I just didn't know there was a default 😄